benchmarks/cli.py

```diff
@@ -0,0 +1,48 @@
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Benchmarks for DataLad CLI"""
+
+import os
+import sys
+import os.path as osp
+
+from subprocess import call
+from .common import SuprocBenchmarks
+
+
+class startup(SuprocBenchmarks):
+    """
+    Benchmarks for datalad command startup
+    """
+
+    def setup(self):
+        # we need to prepare/adjust PATH to point to installed datalad
+        # We will base it on taking sys.executable
+        python_path = osp.dirname(sys.executable)
+        self.env = os.environ.copy()
+        self.env['PATH'] = '%s:%s' % (python_path, self.env.get('PATH', ''))
```

Comment:
I see that it is a copy/paste from the core's `setup`.  Ideally we should avoid that. May be `setup` in SuprocBenchmarks should just do that for all benchmarks?

And even though it is a copy, and probably benchmarks code is *nix specific ATM, but might be worth using `os.pathsep` instead of hardcoded `:`.  

```suggestion
        self.env['PATH'] = f'{python_path}{os.pathsep}{self.env.get("PATH", "")}')
```
assuming that `os` is imported above.

Reply:
I am not sure what these lines do. I just copied them to have something working. I am fine to revert the entire PR after the CLI is merged. Will file an issue.